client/src/components/CRM/Navigation.tsx

```diff
@@ -25,244 +26,462 @@ const Navigation = () => {
+        <div
+          className={
+            showMobileMenu
+              ? 'w-full h-full absolute z-40  transform  translate-x-0 '
+              : '   w-full h-full absolute z-40  transform -translate-x-full'
+          }
+          id="mobile-nav"
+        >
+          <div
+            className="bg-gray-800 opacity-50 absolute h-full w-full md:hidden"
+            onClick={() => {
+              setShowMobileMenu(!showMobileMenu);
+              setShowTaskMenu(false);
+            }}
+          />
           <div className="absolute z-40 sm:relative w-[14rem] md:w-96 shadow bg-zinc-900 md:hidden transition duration-150 ease-in-out h-full">
-            <div className="flex flex-col justify-between h-full w-full">
+            <div className="flex flex-col justify-between h-full w-full overflow-auto">
               <div>
                 <div className="flex items-center justify-between px-6">
                   <div className="h-12  w-full flex flex-col ml-6 items-center ">
                     <button className="flex flex-col  bg-transparent border-transparent justify-center items-center">
-                      <img src={logo} className="mt-[5rem] w-[6rem] " alt="/"/>
+                      <img src={logo} className="mt-[5rem] w-[6rem] " alt="/" />
                     </button>
                   </div>
-                  <button id="closeSideBar" className="flex items-center justify-center h-10 w-10" onClick={() => setShowMobileMenu(!showMobileMenu)}>
-                    <svg xmlns="http://www.w3.org/2000/svg" className="icon icon-tabler icon-tabler-x" width={20} height={20} viewBox="0 0 24 24" strokeWidth="1.5" stroke="white" fill="none" strokeLinecap="round" strokeLinejoin="round">
+                  <button
+                    id="closeSideBar"
+                    className="flex items-center justify-center h-10 w-10"
+                    onClick={() => {
+                      setShowMobileMenu(!showMobileMenu);
+                      setShowTaskMenu(false);
+                    }}
+                  >
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      className="icon icon-tabler icon-tabler-x"
+                      width={20}
+                      height={20}
+                      viewBox="0 0 24 24"
+                      strokeWidth="1.5"
+                      stroke="white"
+                      fill="none"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                    >
                       <path stroke="none" d="M0 0h24v24H0z" />
                       <line x1={18} y1={6} x2={6} y2={18} />
                       <line x1={6} y1={6} x2={18} y2={18} />
                     </svg>
                   </button>
                 </div>
-                <ul className=" space-y-7 mt-30  flex flex-col items-center  ">
+                <ul className=" space-y-7 flex flex-col items-center  ">
```

Comment:
To have some spacing at the bottom of the list (so that the last icon isn't resting on the bottom of the page, could you add the `mb-7` class to this component.

Reply:
Are you referring to the task icon? There is quite a bit of spacing there already

Reply:
@henrykaus Changes have been made, lmk if it looks good!